src/frontend-proxy.ts

```diff
@@ -92,9 +92,13 @@ export async function frontendProxy(
       return null
   }
 
+  const probablyMobile =
+    (request.headers.get('user-agent') || '').indexOf('Mobi') > -1
```

Comment:
```suggestion
  if  ((request.headers.get('user-agent') ?? '').indexOf('Mobi') > -1) {
    return await fetchBackend({ ...config, pathPrefix: config.instance, useFrontend: true, request })
  }
```

It's better to send them to the new frontend without setting a cookie (thus a new dice will be rolled when we want to deactivate this feature due to a bug or something -> otherwise all mobile users will always be redirected to the new frontend)